README.md

```diff
@@ -36,6 +36,12 @@ If you wish to use the PyTorch bindings, then you can use the `ml` flag to insta
 pip install darwin-py[ml]
 ```
 
+If you wish to use LongVideo frame extraction, then you can use the `ocv` flag to install all the additional requirements
```

Comment:
LongVideo is the internal name. Maybe we have some customer-facing name?

Reply:
Is there anything we are calling it for customers?